src/components/ChatBotBody/ChatBotBody.tsx

```diff
@@ -59,7 +59,17 @@ const ChatBotBody = ({
 	const bodyStyle: CSSProperties = {
 		...styles?.bodyStyle,
 		scrollbarWidth: settings.chatWindow?.showScrollbar ? "auto" : "none",
-	}
+	};
+
+	// styles for typing indicator
+	const rcbDot: CSSProperties = {
+		width: "8px",
+		height: "8px",
+		borderRadius: "50%",
+		backgroundColor: "#ccc",
+		marginRight: "4px",
+		animation: "rcb-typing 1s infinite"
```

Comment:
These are currently present in the css file as well, so there is no need to redefine them. The purpose of having `rcbDotStyle` is to expose it as a style prop to users, similar to the other style props here: https://react-chatbotify.com/docs/api/styles